book/modules/module5-exercises.tex

```diff
@@ -208,7 +208,7 @@
 					\[
 						\Comp_{\vec v}(\vec a) = \frac{\vec
 						a \cdot \vec v}{\vec v \cdot
-						\vec v}\vec u = \frac{2}{2}\vec
+						\vec v}\vec v = \frac{2}{2}\vec
 						v = \mat{0\\1\\1}.
```

Comment:
Shouldn't this vector be in R^2?

Reply:
![image](https://user-images.githubusercontent.com/63186767/96324968-7ab33800-0ff2-11eb-9453-16ca9761577f.png)
Sorry about that. This is the typo in the answer of Question 7